cudamapper/src/overlapper_triggered.cu

```diff
@@ -133,6 +133,28 @@ struct FuseOverlapOp
     }
 };
 
+struct FilterOverlapOp
```

Comment:
Can this be a lambda?

Reply:
Ok, I see this is a widely used pattern in `overlapper_triggered`, no need to take care of it in this PR